src/multi_select.coffee

```diff
@@ -15,6 +15,11 @@ Ember.Widgets.MultiSelectOptionView = Ember.View.extend
     @notifyPropertyChange 'label'
   , 'context', 'labelPath'
 
+Ember.Widgets.MultiSelectTooltipItemView = Ember.Widgets.SelectTooltipOptionView.extend
+  processDropDownShown: ->
+    @_super()
+    @get('controller').focusTextField()
```

Comment:
Not sure, but that's not a new change in this commit. This is just adding the same from the regular view to the tooltip view